### DISCUSSION TITLE
Opt-Out of being included in Feed

### DISCUSSION THREAD

**Discussion by willhughes-au** [community/community#16122] [Other Feature Feedback, Questions, & Ideas]
Github Terms of Service mean that I can't have multiple Github accounts. 
This means that my personal use of Github is tied to my Work use of Github. 

Because of this, my account is linked to my employer's organisations.  Through Feed I can see all the activity that my colleagues are doing, including things that are obviously non-work related.  

I do not want Feed, except for people I am specifically following. (i.e no Algorithmic Feed, as [already discussed](https://github.com/github/feedback/discussions/13131))

I also very specifically do not want my data used in other's Feed, unless they are specifically following me.   I'd also like the ability to opt out of having Followers, too. 

I'm not here on Github for social networking, stop forcing this.

**Comment by afshimono** [comment]
I am in the same situation. Really need to disable that new feed from my account ASAP.

**Comment by alanxone** [comment]
It has been widely discussed in #13131 and #13154, with the highest votes to urge GitHub team stop/adjust what they are doing, unfortunately, still no positive feedback from them.

**Comment by siminapasat** [comment]
> I also very specifically do not want my data used in other's Feed, unless they are specifically following me. I'd also like the ability to opt out of having Followers, too.

We've heard this and iterating on future versions of what following means, what gets displayed, and the feed overall. Thank you for your continued feedback. 🙏
